cmd/cleanup.go

```diff
@@ -8,8 +8,8 @@ import (
 
 var cleanupCmd = &cobra.Command{
 	Use:   "cleanup [subdomain]",
-	Short: "cleanup a tunnel that shouldn't be open",
-	Long:  `cleanup a tunnel that shouldn't be open that is associated to the given subdomain`,
+	Short: "Cleanup a tunnel that shouldn't be open",
```

Comment:
How do we define this without using the word in the definition?

"Recover the resources for a crashed tunnel"
"If your tunnel is open, but you're not really using it, use punch cleanup to close it"
"Force a tunnel to close, even if it's not really open"

As a user I have no idea what "cleanup" means in this context.